lib/iris/tests/unit/fileformats/pp/test_PPField.py

```diff
@@ -71,19 +94,7 @@ def test_float64(self):
         # Tests down-casting of >f8 data to >f4.
 
         def field_checksum(data):
-            field = TestPPField()
-            field.dummy1 = 0
-            field.dummy2 = 0
-            field.dummy3 = 0
-            field.dummy4 = 0
-            field.lbtim = 0
-            field.lblrec = 0
-            field.lbrow = 0
-            field.lbext = 0
-            field.lbpack = 0
-            field.lbuser = 0
-            field.brsvd = 0
-            field.bdatum = 0
+            field = TestPPField()._ready_for_save()
```

Comment:
I know it's nit-picky, but why make `_ready_for_save` a private method if you are calling it externally?